client/src/app/home/home.component.html

```diff
@@ -1,7 +1,14 @@
 <div class="flex-row">
-  <div class="flex-1" fxFlex.gt-sm="80" fxFlexOffset.gt-sm="10">
```

Comment:
The `fxFlex` things were clearly not doing real work here, but I do wonder: did you put something in that would address what they used to address? Or, do we just not worry about that?

Reply:
Yeah the new `max-width` for the main container does something similar but not the same, limiting the width on larger screens.